common/web/types/src/package/kps-file.ts

```diff
@@ -94,7 +94,7 @@ export interface KpsFileKeyboard {
   version: string;
   oSKFont?: string;
   displayFont?: string;
-  rtl?: boolean;
+  rTL?: string;
```

Comment:
What sort of convention is this?  I'd rather keep it purely lower-cased.  Also not a fan of `oSKFont`.  (Title-case for an abbrevation is... okay if necessary.  I'm generally not a fan of mixed-case within a coding abbreviation unless it causes issues parsing something in camelCase or similar.)

I checked the referenced issue, but there's no detail there for me to compare against.

Reply:
It sounds like there may be some pre-established spec that mandates this... but there was no link for me to follow and verify that.  I'll definitely acquiesce if this is the case.

Reply:
We've discussed this before. It's a "feature" of the xml2js implementation and options selected which I am still considering winding back, see #7353. I reckon we should call it dONKEYCase (or aSSCase but that's a little dangerous).

Reply:
Ah, so not exactly a "spec", but it's a pre-established thing we have to put up with for now b/c triage.  :+1:  Good to have the reference, at least.